### ISSUE TITLE
Material UI Lite Themes not working

### ISSUE THREAD

**Issue by MioBlanco** [JFormDesigner/FlatLaf#89]
When choosing a Theme, that is a material ui theme the following exception is thrown:

```
15:42:20,187 ERROR AbstractLeaUncaughtExceptionHandler:435 - Uncaught exception on thread: Thread[main,5,]
java.lang.NullPointerException
	at java.io.Reader.<init>(Reader.java:78)
	at java.io.InputStreamReader.<init>(InputStreamReader.java:113)
	at com.formdev.flatlaf.IntelliJTheme.<init>(IntelliJTheme.java:112)
	at com.formdev.flatlaf.intellijthemes.materialthemeuilite.Utils.loadTheme(Utils.java:35)
	at com.formdev.flatlaf.intellijthemes.materialthemeuilite.FlatGitHubIJTheme.<init>(FlatGitHubIJTheme.java:36)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at java.lang.Class.newInstance(Class.java:442)
	at javax.swing.UIManager.setLookAndFeel(UIManager.java:583)
```

We've installed the theme via:

```java
UIManager.setLookAndFeel("com.formdev.flatlaf.intellijthemes.materialthemeuilite.FlatGitHubIJTheme");
```

Using the static `install` method doesn't work either, but doesn't throw an exception.

This only happens with the material themes. The other ones work fine with both methods. Is there anything we are overlooking?

**Comment by DevCharly** [comment]
Thanks, fixed in master (see PR #88)

Could you test the latest `0.33-SNAPSHOT` snapshot?
https://github.com/JFormDesigner/FlatLaf#snapshots

**Comment by DevCharly** [comment]
fixed in 0.33
